File: VB/dxSample_CustomSimpleButton/Custom Button/CustomSimpleButtonViewInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors.ViewInfo

Namespace dxSample_CustomSimpleButton.Custom_Button
    Public Class CustomSimpleButtonViewInfo
        Inherits SimpleButtonViewInfo

        Public Sub New(ByVal owner As DevExpress.XtraEditors.SimpleButton)
            MyBase.New(owner)
        End Sub
        Protected Overrides Function GetButtonPainter() As DevExpress.XtraEditors.Drawing.EditorButtonPainter
            Return New CustomSkinEditorButtonPainter(LookAndFeel.ActiveLookAndFeel, Owner)
        End Function
    End Class
End Namespace

File: CS/dxSample_CustomSimpleButton/Custom Button/CustomSimpleButtonViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors.ViewInfo;

namespace dxSample_CustomSimpleButton.Custom_Button {
    public class CustomSimpleButtonViewInfo : SimpleButtonViewInfo {
        public CustomSimpleButtonViewInfo(DevExpress.XtraEditors.SimpleButton owner)
            : base(owner) {
        }
        protected override DevExpress.XtraEditors.Drawing.EditorButtonPainter GetButtonPainter() {
            return new CustomSkinEditorButtonPainter(LookAndFeel.ActiveLookAndFeel, Owner);
        }
    }
}
